src/main/data_structure/tuple_exercise.py

```diff
@@ -57,4 +57,18 @@ def tuple_to_string(self):
 
     # sort a tuple by its float element.
     def sort_tuple_by_float_element(self):
-        return tuple(sorted(self.tuple, key=lambda x: float(x[1]), reverse=True))
\ No newline at end of file
+        return tuple(sorted(self.tuple, key=lambda x: float(x[1]), reverse=True))
+
+    # Count and average Best three tutorial marks
+    def average_tutorial_marks(self):
```

Comment:
function name should be verb in this case you can use calculate_average_tutorial_marks instead of average_tutorial_marks